### PR TITLE
trigger release build on Git tag push

### PR DESCRIPTION
**Pull request by brennerm**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -19,6 +21,8 @@ jobs:
           GH_BUILD_PASSWORD: ${{ secrets.GH_BUILD_PASSWORD }}
       run: |
         TAG=$(date +%Y-%m-%d)-$(git rev-parse --short HEAD)
+        [[ "${{ github.ref }}" == "refs/tags/"* ]] && TAG=$(echo "${{ github.ref }}" | sed 's/refs\/tags\/[v]//')
```
Comment (ArifAlam) [inline .github/workflows/build.yml:24]: Just want to confirm lines similar to line 24 will only get run on tagged commits and that untagged commits will use the TAG from line 23.

Comment (brennerm) [reply .github/workflows/build.yml:24]: Yup, that's the intention. :+1:

**Comment by blogic** [comment]
the expected PR will contain within its patches the following format
$prefix: $descprition
$explanation
Signed-off-by: $name <$email>

**Comment by brennerm** [comment]
@blogic Done :+1:

**Comment by RickSommerville** [comment]
Merged, thanks

**Pull request closed without merging**